agent/src/softwarecontaineragent.cpp

```diff
@@ -297,6 +297,22 @@ bool SoftwareContainerAgent::execute(ContainerID containerID,
         return false;
     }
 
+    // We want to always apply any default capabilities we have. If the container is in READY state,
+    // that means that its gateways have been configured. The only way to configure the gateways
+    // from the agent is through setCapabilities - which sets the default caps also.
+    //
+    // If it has not been set, then we use a call without arguments to setCapabilities to set it up,
+    // since we then should get only the default ones.
```

Comment:
For multi line comments it would be more consistent with the rest of the code to use:
/*
 *
 */